src/components/creator/Selection.tsx

```diff
@@ -78,17 +128,22 @@ const CharacterCards = () => {
 export const CreatorSelection = () => {
 	const { t } = useTranslation("creator")
 
+	useEffect(() => {
+		window.scrollTo(0, 0)
+	  }, [])
+
 	return (
 		<>
 			<Header />
 			<ChallengeInProgressDialog />
-			<Container className={styles.selection} maxWidth={"xl"}>
+			<Container className={styles['selection']} maxWidth={"xl"}>
```

Comment:
¿Es necesario en esta pantalla un maxWidth? Queda así en pantallas grandes:

![imagen](https://github.com/Program-AR/pilas-bloques-react/assets/48812037/829be35f-edee-4e9a-88e4-224a5df1948c)

```suggestion
			<Container className={styles['selection']} maxWidth={false}>
```

Reply:
es la vieja herencia recibida ajajajaj!!! ya estaba asi cuando tomé el issue jajaja! Ahora lo commiteo. Gracias @dlopezalvas